templates/gem/<%= gem_name %>.gemspec

```diff
@@ -0,0 +1,33 @@
+=begin
+Copyright <%= copyright %>
+
+See the file LICENSE for copying permission.
+=end
+
+# -*- encoding: utf-8 -*-
+
+lib = File.expand_path('../lib', __FILE__)
+$LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
+require 'pre-commit/<%= name %>/version'
+
+Gem::Specification.new do |s|
+  s.name     = "<%= gem_name %>"
+  s.version  = PreCommit::<%= name.capitalize %>::VERSION
+  s.author   = "<%= author %>"
+  s.email    = "<%= email %>"
+  s.homepage = "https://github.com/pre-commit-plugins/<%= gem_name %>"
```

Comment:
I did not make `pre-commit-plugins` configurable, we already have to pass on to many options to the new command, it might also encourage using this common repository instead of personal ones